src/giftCards/GiftCardsList/GiftCardsListTable/GiftCardsListTable.tsx

```diff
@@ -7,50 +7,48 @@ import {
 } from "@material-ui/core";
 import HorizontalSpacer from "@saleor/apps/components/HorizontalSpacer";
 import Checkbox from "@saleor/components/Checkbox";
+import ContentOrSkeleton from "@saleor/components/ContentOrSkeleton";
 import DeleteIconButton from "@saleor/components/DeleteIconButton";
 import Link from "@saleor/components/Link";
 import ResponsiveTable from "@saleor/components/ResponsiveTable";
 import StatusChip from "@saleor/components/StatusChip";
 import { StatusType } from "@saleor/components/StatusChip/types";
 import { customerUrl } from "@saleor/customers/urls";
-import { giftCardPath } from "@saleor/giftCards/urls";
+import { giftCardUrl } from "@saleor/giftCards/urls";
 import useNavigator from "@saleor/hooks/useNavigator";
 import { renderCollection } from "@saleor/misc";
 import { productUrl } from "@saleor/products/urls";
 import React, { useContext } from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 
-import { GiftCardsListContext } from "../GiftCardsListProvider";
 import { giftCardsListTableMessages as messages } from "../messages";
+import { GiftCardsListContext } from "../providers/GiftCardsListProvider";
 import { useTableStyles as useStyles } from "../styles";
 import GiftCardsListTableFooter from "./GiftCardsListTableFooter";
 import GiftCardsListTableHeader from "./GiftCardsListTableHeader";
 
-const numberOfColumns = 7;
 const PLACEHOLDER = "-";
 
-// interface GiftCardsListTableProps {}
-
-const GiftCardsListTable: React.FC = ({}) => {
+const GiftCardsListTable: React.FC = () => {
   const intl = useIntl();
   const classes = useStyles({});
   const navigate = useNavigator();
-  const { toggle, isSelected, giftCards } = useContext(GiftCardsListContext);
-
-  const commonTableProps = {
-    numberOfColumns
-    // TEMP
-    // disabled
-  };
+  const {
+    toggle,
+    isSelected,
+    giftCards,
+    numberOfColumns,
+    loading
+  } = useContext(GiftCardsListContext);
```

Comment:
Didn't we want to export a hook: `useGiftCardBulkActions` that hooks into this context?